util/util.go

```diff
@@ -89,42 +94,46 @@ func (s server) GetEventTeams(ctx context.Context, in *pb.GetEventTeamsRequest)
 			SolvedChallenges: t.SolvedChallenges,
 		})
 	}
-
+	log.Printf("Get Teams for the Event %s", in.EventTag)
 	return &pb.GetEventTeamsResponse{Teams: teams}, nil
 }
 
 func (s server) UpdateEventFinishDate(ctx context.Context, in *pb.UpdateEventRequest) (*pb.UpdateResponse, error) {
 	result, err := s.store.UpdateEventFinishDate(in)
 	if err != nil {
+		log.Printf("ERR: Error Update Event %s finish time: %s", in.EventId, err.Error())
 		return &pb.UpdateResponse{ErrorMessage: err.Error()}, nil
 	}
+	log.Printf("Event %s Stopped", in.EventId)
 	return &pb.UpdateResponse{Message: result}, nil
 }
 
 func (s server) UpdateTeamSolvedChallenge(ctx context.Context, in *pb.UpdateTeamSolvedChallengeRequest) (*pb.UpdateResponse, error) {
 	result, err := s.store.UpdateTeamSolvedChallenge(in)
 	if err != nil {
+		log.Printf("ERR: Error Update team %s solve challenge: %s", in.TeamId, err.Error())
 		return &pb.UpdateResponse{ErrorMessage: err.Error()}, nil
 	}
+	log.Printf("Team %s solved %s challenge", in.TeamId, in.Tag)
 	return &pb.UpdateResponse{Message: result}, nil
 }
 
 func (s server) UpdateTeamLastAccess(ctx context.Context, in *pb.UpdateTeamLastAccessRequest) (*pb.UpdateResponse, error) {
 	result, err := s.store.UpdateTeamLastAccess(in)
 	if err != nil {
+		log.Printf("ERR: Error Update team %s last access: %s", in.TeamId, err.Error())
 		return &pb.UpdateResponse{ErrorMessage: err.Error()}, nil
 	}
 	return &pb.UpdateResponse{Message: result}, nil
 }
 
-func GetCreds() (credentials.TransportCredentials,error) {
+func GetCreds(conf *model.Config) (credentials.TransportCredentials,error) {
 	log.Printf("Preparing credentials for RPC")
-	// todo: change environment variables into configuration
-	// add handling functionality
+
 	certificateProps := certificate{
-		cPath:    			os.Getenv("CERT"),
-		cKeyPath: 			os.Getenv("CERT_KEY"),
-		caPath:             os.Getenv("CA"),
+		cPath:    			conf.TLS.CertFile,
+		cKeyPath: 			conf.TLS.CertKey,
+		caPath:             conf.TLS.CAFile,
```

Comment:
Indentation. You can actually setup auto indentation using Goland, we can have a call about it if you want.